azure-functions/acbs-function/mappings/facility/facility-master.js

```diff
@@ -56,7 +56,7 @@ const facilityMaster = (deal, facility, acbsData, acbsReference) => {
     facilityIdentifier: facility.facilitySnapshot.ukefFacilityId.padStart(10, 0),
     portfolioIdentifier: CONSTANTS.FACILITY.PORTFOLIO.E1,
     dealBorrowerIdentifier: acbsData.parties.exporter.partyIdentifier,
-    maximumLiability: helpers.getMaximumLiability(facility.facilitySnapshot),
+    maximumLiability: helpers.getMaximumLiability(facility),
```

Comment:
does the `facility` have the dealType in it? I've seen that in the function we are checking
```
const ukefExposure = dealType === CONSTANTS.PRODUCT.TYPE.GEF
    ? amount * 0.10
    : amount;
```

but if it doesn't have the dealType, then it will always be false

Reply:
Above snippet is from `getLoanMaximumLiability` helper function, which has second argument as `dealType`.
` getMaximumLiability` only has one argument as `facility` object.